src/lib.rs

```diff
@@ -313,35 +313,75 @@ mod contract {
     }
 
     #[no_mangle]
-    pub extern "C" fn deposit() {
-        EthConnectorContract::new().deposit()
+    pub extern "C" fn deposit_near() {
```

Comment:
Question: What are we writing these exports ourselves instead of using the near sdk macros like the contracts in https://github.com/near/core-contracts ? Is it for performance reasons?